libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1116,10 +1116,12 @@ function baro:get_pressure() end
 ---@class serial
 serial = {}
 
--- desc
----@param protocol integer
----@return AP_HAL__UARTDriver_ud
-function serial:find_serial(protocol) end
+-- Returns the UART instance that allows connections from scripts (those with SERIALx_PROTOCOL = 28`).
+-- For port_number = 0, returns first such UART, second for port_number = 1, and so on.
+-- If the instance is not found, returns nil.
+---@param port_number integer -- the 0-based index of the UART instance to return.
+---@return AP_HAL__UARTDriver_ud -- the (port_num)-th UART instance available for scripting, or nil if none.
+function serial:find_serial(port_number) end
```

Comment:
maybe "instance" would be better than "port_number" ?

Reply:
Yes. Made a commit.